Wabbajack.Lib/ModListRegistry/ModListMetadata.cs

```diff
@@ -31,6 +32,9 @@ public class ModlistMetadata
         [JsonProperty("game")]
         public Game Game { get; set; }
 
+        [JsonIgnore]
```

Comment:
I think this should just be a getter property like
`public string GameName => this.Game.ToDescriptionString();`

Rather than an actual field we have to set, that could be potentially desynced (Skyrim `Game` enum, /w `GameName` set to "Fallout")

Reply:
If you want the fancy functionality of using description if it's set, otherwise fall back to `ToString` I'd just define and extension function to do that logic `DescriptionOrToString`